WordPress/Classes/Services/EditorSettingsService.swift

```diff
@@ -65,47 +65,50 @@ import Foundation
         }, failure: failure)
     }
 
-    func syncEditorSettingsForAllBlogs() {
-        let blogService = BlogService(managedObjectContext: managedObjectContext)
-        guard let blogs = blogService.blogsForAllAccounts() as? [Blog] else {
+    /// This method is intended to be called only after the db 87 to 88 migration
+    ///
+    func postAppWideEditorSettingToRemoteForAllBlogsAfterMigration(database: KeyValueDatabase = UserDefaults.standard) {
```

Comment:
This is OK but a bit verbose, how about `migrateGlobalSettingToRemote`. And since the migration calling is already looking up the value, why not pass a `Bool` with the setting instead of the database?

Reply:
> And since the migration calling is already looking up the value, why not pass a Bool 

That sounds good! I was thinking about something like that on https://github.com/wordpress-mobile/WordPress-iOS/pull/12276/files#r311444419 👍